### PR TITLE
[doc] Minor fix in intro of Sec HW Design Guideline

### PR DESCRIPTION
**Pull request by johannheyszl**
Fixed intro re SCA/DPA

### CODE REVIEW

File: doc/security/implementation_guidelines/hardware/README.md [hunk 1]
```diff
@@ -69,12 +69,13 @@ redundancy/detectors) and requires only modest numbers of traces.
...
-very powerful because they are able to "divide and conquer" an AES operation
-(regardless of key size) into its 8-bit S-Boxes and enumerate all 256 possible
-values to evaluate hypotheses. Evaluating/processing information in 32-bit
-quanta (or larger) will make these kinds of enumerations much more difficult;
-operating on a single bit at a time makes them almost trivial.
+work with "small" subsets of bits of sensitive information. All side channel attacks
+use a strategy of "divide and conquer" to recover *secret material* e.g. word by
+word or byte by byte. Making sure to evaluate/process information in 32-bit quanta
+or larger will make this more difficult. Attacks like DPA are especially powerful
+because on top of a "divide and conquer" where they address 8-bit intermediate
```
Comment (vogelpi) [inline doc/security/implementation_guidelines/hardware/README.md:76]: Nit: where they e.g. address 8-bit intermediate... (AES is just one common DPA attack target but what the text describes holds for any DPA target).

**Review by vogelpi — APPROVED** [review]
LGTM

File: doc/security/implementation_guidelines/hardware/README.md (hunk 1 above)
Comment (johannheyszl) [reply doc/security/implementation_guidelines/hardware/README.md]: thx , fixed :)

**Review by vrozic — APPROVED** [review]

**Comment by cdgori** [comment]
LGTM! Thanks for making this both more clear and more general.

**Comment by vogelpi** [comment]
Congrats to your first PR @johannheyszl ;-)

**Pull request merged**